.travis.yml

```diff
@@ -20,23 +20,21 @@ jobs:
   include:
 # the secure configurations in env: is for BINTRAY_USER=<USER> and BINTRAY_KEY=<KEY> properties
 # which will be used for publishing artifacts to snapshot repository
-    - stage: snapshot
-      if: branch = master
-      env:
-         - secure: "EJGO5B4XxT6NpZ4+St4EG/Y3+UmQ2XNkbnJz/5vv7HZziC4OXFVFCigOsjqqRVjJaWmDmYf1a/U7QwJa+esgA9oBchg+DVWYeqkMOEbZkEJzBgHeA5Bmgp+3Q5/2ouOe6M2cUFhSNXN+lZaLri8xYis25ndK6E0Y6MIQ7CbCNyhA/edIIHSttJNHoGkaVxkfEDWNvIvdUbxt7MTTdAGIvT+Xj/EF8clT349N5UfZ8DdYy8Q6i4TKiy5557dj6nfegbD6qD0mGxWt8dqKBSIqmPMoiCjGsy+3LyfeMNhZiNji+7LRmwJtWD87fewlYeU5xkKZIaxXlCnd9JPrkIFH62FzXctn0wc18mDTD+c4mnc2efKHHrmvRcrvQrRD447iFt17lEMCROriHVZsB4m6igxrEg0i2paCBajMqhQG5LEwSqU7D8CQaW+ffC2+9agFR0a6uhW7n9GahU2AOVf9RhvLU7yFWtVJltOP2mBY0s8Oowxzy5OXSfqjxwSJeKKGgWVaTbBY14Y1Y2pCx9CuInLswz4oIO3LqHHfKJasKwHp9nHRSa6flh8JFR8+jGA7vo1MqGAPH3N4rrMQqpYs9DjYD50TPAh2h4asQj5gSjtYkKpvEu58dRRRzansjtSpiwo6TNfaVp9jhzYI3O3vnssyDyJvHjHjCnrvCB/KYrU="
-         - secure: "NdkraKeGWu2OLxxHEjxn09GdVaXeKxaSrpNuTMc38Bnq9O2msXgb2jiEBsq4Z23j8cxG6Y4JJ0ZAOQwhuB3Tfh6UwGxSf8CxAqplD9uy1VRrAT0gXneIjaAwlOITy5IQfP0bY0o+IwYSefLEKnR46YLxd4zHckgneq5SWD9bUZEByM7rcYZArpaHK/fBQ+0aM0SsYY0vHX/6ivA9f9oWIva2pnWgOXPABbgrN7Xah+a+XVAV3Qe2vBA0MAl0pA1S9PWWfZNbVAO+CSR6d4+Rdvn4tQsM5Si5p9hWjmBi22kkDMJtLAhKHUnutoHPVMH6ekQneakshhiVuhMzAJZoMI626P5GkiyN3OVkv+AlAGEUZdJnXlsx51veIAvQh1iZZhpJ5Cd5R8H7dQrgIFiios+fRLlD8VbCwv8NAL366khBkQjoW2RyF+Xl1Y1J7Yd9C7lsg/Ktm7PNIVNRsHjMu1msQHNNsn0VmVbH2IZo20u8giKhiUVcAznctj5kmleCBgszTlRj07pxMQggcLrK0d0y1gU9Gg/pOubcsHNzQ/pLWVjep1vuJ+wyW64BtgQuFvPK97eA95P5b3j3MpalhFUKPF+mVdqXNQjcJrYOwtRagq4CkPsg3fKr7+HMzUCHwo69eqJfgPCOR9NYLSMPzhKAl/YrdphH46wPuEiOopU="
-      install: ./gradlew assemble
-      script: ./gradlew check publish -PpublishUrl=jcenterSnapshot -PpublishUsername=$BINTRAY_USER -PpublishPassword=$BINTRAY_KEY
-      after_success:
-           - bash <(curl -s https://codecov.io/bash) -t ccceafaf-7c60-4a02-9165-480174b535a2
-
     - stage: build
       if: type IN (push, pull_request)
       install: ./gradlew assemble
       script: ./gradlew check
       after_success:
            - bash <(curl -s https://codecov.io/bash) -t ccceafaf-7c60-4a02-9165-480174b535a2
 
+    - stage: snapshot
+      if: branch = master
```

Comment:
Can you please make it `if: branch = master AND type != pull_request`. 
We don't want every commit to pull request, publish snapshots to jcenter.

Reply:
No, please check the commits to PRs,  after your PR went into master, every commit has 2 build stages. for ex : this build https://travis-ci.org/pravega/pravega/builds/382607193?utm_source=github_status&utm_medium=notification
from PR https://github.com/pravega/pravega/pull/2618.

This is not yet merged into master, but it tries to publish snapshots to jcenter.

Reply:
*  Every PR from fork branch to master of pravega, has event type `pull_request` , (continuous-integration/travis-ci/pr)  and the branch will be taken as master.
*  By default the event type of `pull_request` or `push` is considered  in the build stages, unless some condition is specified in `travis.yml`.

Reply:
I think you you are right. The build stage is applying for PRs merging to master as well. Thanks for finding it and I will fix the issue.